lib/Utils/Logging.pm

```diff
@@ -374,4 +375,36 @@ sub export_logs_desktop {
     }
 }
 
+# I am not sure if this should even be here
+my %avc_record = (
+    start => 0,
+    end => undef
+);
+
+=head2 record_avc_selinux_alerts
+
+List AVCs that have been recorded during a runtime of a test module that executes this function
+
+=cut
+
+sub record_avc_selinux_alerts {
+
+    if ((current_console() !~ /root|log/) || (script_run('test -f /var/log/audit/audit.log') != 0)) {
+        return;
+    }
+
+    my @logged = split(/\n/, script_output('ausearch -m avc -r', timeout => 300, proceed_on_failure => 1));
```

Comment:
I think we could change this to:
```suggestion
    my @logged = split(/\n/, script_output('ausearch -m avc,user_avc,selinux_err,user_selinux_err', timeout => 300, proceed_on_failure => 1));
```

Reply:
Yep, that's where I'm coming from, added it https://openqa.opensuse.org/tests/5064686#step/snapper_zypp/92 life is good